README.md

```diff
@@ -1 +1,59 @@
+There are multiple ways to achieve centralized monitoring in AWS. If you do not need to use the CloudWatch APIs, you can set up a
+multi-account dashboard in the CloudWatch console, but you'll have to manually select and repeat this process for relevant metrics and resources. The advantage to using this script is that it will enable you to centralize, collect and report on any (standard) AWS metric for any resource you have access to.
+
+### Ongoing maintenance
+By leveraging CloudFormation StackSets and AWS Organizations you greatly reduce your ongoing burden to maintain centralized monitoring. As you add and delete accounts in your organization, new Stacks creating links to your centralized sink will automatically be created and destroyed on a per-region, per-account basis.
+
+### Prerequisites:
+- [`Python3`](https://www.python.org/downloads/) (We're using 3.11)
+- `Pip3` (Comes standard with Python > 3.4)
+- Optional, but highly recommended `VirtualEnv`
+  - `pip3 install virtualenv`
+- AWS CLI Credentials for your management account
+
+### Setup your environment
+- If you have `virtualenv` installed:
+  - `source env_setup.sh`
+- If you don't have `virtualenv`:
+  - `pip3 install -r requirements.txt`
+
+### Run the script
+- `python3 create_links_sinks.py --sink-name SINK-NAME --regions "region1,region2,region3" --profile AWS-CLI-PROFILE --organization ORGANIZATION --organization-unit ORGANIZATION-UNIT --excluded-accounts "accountID1,accountID2,accountID3"`
+  - `sink-name`: Human-friendly name for your sink and StackSet
+  - `regions`: _Optional_ Double quoted, comma separated list of regions to create a sink and StackSet in
+    - **Do not add spaces between regions**
+    - If not specified, uses Regions enabled by default in an AWS Account ("us-east-2", "us-east-1", "us-west-1", "us-west-2",
+		"ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2",
+		"ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3",
+		"eu-north-1", "sa-east-1")
+  - `profile`: AWS CLI Credentials profile. If not specified, uses the default profile
+  - `organization`: Unique identifier for your organization
+    - Navigate to [Organizations](https://us-east-1.console.aws.amazon.com/organizations/v2/home/accounts), click on Root, and get the string starting with o- from your organization ARN
+    - arn:aws:organizations::01234567890:root/o-oooo1oo1oo/r-root -> o-oooo1oo1oo
+  - `organization-unit`: Organization unit that you want to centralize monitoring for
+    - We recommend root (arn:aws:organizations::01234567890:root/o-oooo1oo1oo/r-root -> r-root)
+    - Can also be a sub-ou (arn:aws:organizations::01234567890:root/o-oooo1oo1oo/ou-root-ooooo1oo -> ou-root-ooooo1oo)
+  - `excluded-accounts`: _Optional_ Double quoted, comma separated list of accounts to exclude from centralized monitoring
+    - **Suspended accounts MUST be excluded**
+    - **Do not add spaces between excluded accounts**
```

Comment:
Looks like your CLI already does a good job of printing out usage instructions; here's what I get with `./create_links_sinks.py -h` today:

``` 
usage: create_links_sinks.py [-h] --organization ORGANIZATION --organization-unit ORGANIZATION_UNIT [--sink-name SINK_NAME] [--profile PROFILE] [--regions REGIONS]
                             [--excluded-accounts EXCLUDED_ACCOUNTS]

optional arguments:
  -h, --help            show this help message and exit
  --organization ORGANIZATION
                        AWS Organization
  --organization-unit ORGANIZATION_UNIT
                        AWS Organization Unit
  --sink-name SINK_NAME
                        Human-friendly AWS Metric Sink Name, defaults to "Centralized-Monitoring"
  --profile PROFILE     AWS CLI Config Profile Name for the Management Account, uses your default AWS profile if not specified
  --regions REGIONS     Double quoted, comma separated list of regions to create a sink and StackSet in. If not specified, a Sink/StackSet will be created in all default AWS regions
  --excluded-accounts EXCLUDED_ACCOUNTS
                        [Optional] double quoted, comma separated list of accounts to exclude (ex. --exclude "12345678910, 09876543210"). Must exclude suspended accounts
```

imo, you could just beef up the builtin help docs and show the output here in the README if you want, so you don't have to deal with maintaining/formatting the content in two places.